.devops/deploy-pipelines.yml

```
@@ -318,42 +318,150 @@ stages:
+  #       eq(${{parameters.PROD_SKIP_BLUE_DEPLOYMENT}}, False),
+  #       eq(dependencies.Build_PROD_Blue.result, 'Succeeded'),
+  #       in(dependencies.Release.result, 'Succeeded', 'Skipped')
+  #     )
+  #   variables:
+  #     app_version: $[ stageDependencies.Release.make_release.outputs['chart_current_version.appVersion'] ]
+  #   jobs:
+  #     - deployment: "Blue_PROD_deployment"
+  #       displayName: "Blue PROD deployment"
+  #       pool:
+  #         name: pagopa-prod-linux
+  #       environment: 'PROD'
+  #       strategy:
+  #         runOnce:
+  #           deploy:
+  #             steps:
+  #               - checkout: self
+  #                 displayName: "Checkout"
+  #               - task: Bash@3
+  #                 name: update_chart_version
+  #                 displayName: 'Setup helm microservice chart'
+  #                 inputs:
+  #                   targetType: "inline"
+  #                   script: |
+  #                     helm repo add microservice-chart https://pagopa.github.io/aks-microservice-chart-blueprint
+  #                     helm dep build helm
+  #               - template: azure-templates/helm-microservice-chart-deploy.yml
+  #                 parameters:
+  #                   DO_DEPLOY: true
+  #                   DO_BLUE_GREEN_DEPLOY: true
+  #                   ENV: 'PROD'
+  #                   KUBERNETES_SERVICE_CONN: $(PROD_KUBERNETES_SERVICE_CONN)
+  #                   NAMESPACE: ecommerce
+  #                   APP_NAME: $(K8S_IMAGE_REPOSITORY_NAME)
+  #                   VALUE_FILE: "helm/values-prod.yaml"
+  #                   GREEN_VERSION: $(app_version)
+  #                   BLUE_VERSION: $(Build.SourceVersion)
+
+  # - stage: "PROD_Green_WaitForApproval"
+  #   displayName: 'PROD green approval deployment'
+  #   ${{ if eq(parameters['PROD_SKIP_BLUE_DEPLOYMENT'], True) }}:
+  #     dependsOn: Prod_WaitForApproval
+  #     condition: |
+  #       and(
+  #         eq(${{parameters.UAT_PROD_DEPLOY}}, true),
+  #         eq(dependencies.Prod_WaitForApproval.result, 'Succeeded')
+  #       )
+  #   ${{ if eq(parameters['PROD_SKIP_BLUE_DEPLOYMENT'], False) }}:
+  #     dependsOn: Deploy_PROD_Blue
+  #   variables:
+  #     release_url: $[ stageDependencies.Release.make_release.outputs['chart_current_version.releaseUrl'] ]
+  #   jobs:
+  #     - job: Bluegreen_PROD_WaitForApproval
+  #       displayName: Manual green deploy approval
+  #       pool: server
+  #       timeoutInMinutes: 4320 # 3 days
+  #       steps:
+  #         - task: ManualValidation@0
+  #           timeoutInMinutes: 4320 # 3 days
+  #           inputs:
+  #             notifyUsers: $(APPROVE_TOUCHPOINT_MAIL)
+  #             instructions: "Please approve or reject PROD green promotions for release $(release_url)"
+  #             onTimeout: 'reject'
+
+  - stage: "Build_PROD_Green"
+    displayName: 'PROD green Build'
+    dependsOn: [ PROD_Green_WaitForApproval,Release ]
```

```suggestion
    dependsOn: [ Prod_WaitForApproval,Release ]
```
this step should depends from manual approval one